rotkehlchen/tests/api/test_snapshots.py

```diff
@@ -0,0 +1,178 @@
+        """, ('A', 1, 'ADA', '1.00', '178.44'),
+    )
+    cursor.execute(
+        """
+        INSERT INTO "timed_balances" ("category", "time", "currency", "amount", "usd_value") VALUES
+        (?, ?, ?, ?, ?);
+        """, ('A', 1, 'AVAX', '1.00', '87'),
+    )
+    connection.commit()
+
+
+def _populate_db_with_location_data(connection):
+    cursor = connection.cursor()
+    cursor.execute(
+        """
+        INSERT INTO "timed_location_data" ("time", "location", "usd_value") VALUES
+        (?, ?, ?);
+        """, (1, 'A', '100.00'),
+    )
+    cursor.execute(
+        """
+        INSERT INTO "timed_location_data" ("time", "location", "usd_value") VALUES
+        (?, ?, ?);
+        """, (1, 'B', '200.00'),
+    )
+    cursor.execute(
+        """
+        INSERT INTO "timed_location_data" ("time", "location", "usd_value") VALUES
+        (?, ?, ?);
+        """, (1, 'H', '50.00'),
+    )
+    connection.commit()
+
+
+def assert_csv_export_response(response, csv_dir, is_download=False):
+    if is_download:
+        assert response.status_code == HTTPStatus.OK
+    else:
+        assert_proper_response(response)
+        data = response.json()
+        assert data['message'] == ''
+        assert data['result'] is True
+
+    with open(os.path.join(csv_dir, BALANCES_FILENAME), newline='') as csvfile:
+        reader = csv.DictReader(csvfile)
+        count = 0
+        for row in reader:
+            assert len(row) == 4
+            assert row['category'] in (
+                'asset',
+                'liability',
+            )
+            assert row['amount'] is not None
+            assert row['usd_value'] is not None
+            count += 1
+        assert count == 2
+
+    with open(os.path.join(csv_dir, LOCATION_DATA_FILENAME), newline='') as csvfile:
+        reader = csv.DictReader(csvfile)
+        count = 0
+        for row in reader:
+            assert len(row) == 2
+            assert row['location'] in (
+                'external',
+                'kraken',
+                'poloniex',
+                'bittrex',
+                'binance',
+                'bitmex',
+                'coinbase',
+                'total',
+                'banks',
+                'blockchain',
+                'coinbasepro',
+                'gemini',
+                'equities',
+                'realestate'
+                'commodities',
+                'cryptocom'
+                'uniswap',
+                'bitstamp',
+                'binanceus',
+                'bitfinex',
+                'bitcoinde',
+                'iconomi',
+                'kucoin',
+                'balancer',
+                'loopring',
+                'ftx',
+                'nexo',
+                'blockfi',
+                'independentreserve',
+                'gitcoin',
+                'sushiswap',
+                'uphold',
+                'bitpanda',
+                'bisq',
+                'ftxus',
+            )
```

Comment:
You can check that in a much easier way
```suggestion
            assert Location.deserialize(row['location']) is not None
```